src/App.css

```diff
@@ -0,0 +1,42 @@
+#root {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 2rem;
+  text-align: center;
+}
+
+.logo {
+  height: 6em;
+  padding: 1.5em;
+  will-change: filter;
```

Comment:
Is this `will-change` automatically generated? If not, why do we need it?

In general, using `will-change` should be a last resort. In *almost* every scenario we should let the browser decide how to optimise its resources.

Reply:
Part of the template.